File: App_19k.Service/Implementacion/ActividadEconomicaService.cs
using System;
using System.Collections.Generic;
using System.Text;
using App_19k.Domain;
using App_19k.Repository;
using App_19k.Repository.Implementacion;
namespace App_19k.Service.Implementacion
{
    public class ActividadEconomicaService : IActividadEconomicaService
    {

        private IActividadEconomicaRepository ActividadEconomicaRepository;

        public ActividadEconomicaService(IActividadEconomicaRepository ActividadEconomicaRepository)
        {
            this.ActividadEconomicaRepository = ActividadEconomicaRepository;
        }    

        public bool Delete(int id)
        {
            throw new NotImplementedException();
        }

        public ActividadEconomica Get(int id)
        {
            return ActividadEconomicaRepository.Get(id);
        }

     

        public IEnumerable<ActividadEconomica> GetAll()
        {
            return ActividadEconomicaRepository.GetAll();
        }

        public IEnumerable<ActividadEconomica> LstaActEcoXProyecto(int idProyecto)
        {
            return ActividadEconomicaRepository.LstaActEcoXProyecto(idProyecto);
        }

        public bool Save(ActividadEconomica entity)
        {
            return ActividadEconomicaRepository.Save(entity);
        }    

        public bool Update(ActividadEconomica entity)
        {
            return ActividadEconomicaRepository.Update(entity);
        }

       
    }
}

File: src/main/java/com/pe/app19k/service/impl/ActividadEconomicaService.java
package com.pe.app19k.service.impl;

import com.pe.app19k.entity.ActividadEconomica;
import com.pe.app19k.repository.IActividadEconomicaRepository;
import com.pe.app19k.service.IActividadEconomicaService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;
import org.springframework.transaction.annotation.Transactional;

import java.util.List;
import java.util.Optional;

@Service
public class ActividadEconomicaService implements IActividadEconomicaService {
    @Autowired
    private IActividadEconomicaRepository actividadEconomicaRepository;

    @Override
    public ActividadEconomica Insert(ActividadEconomica actividadEconomica) throws Exception {
        return actividadEconomicaRepository.save(actividadEconomica);
    }

    @Override
    public ActividadEconomica Update(ActividadEconomica actividadEconomica) throws Exception {
        return actividadEconomicaRepository.save(actividadEconomica);
    }
    @Transactional(readOnly=true)
    @Override
    public List<ActividadEconomica> FindAll() throws Exception {
        return actividadEconomicaRepository.findAll();
    }
    @Transactional(readOnly=true)
    @Override
    public Optional<ActividadEconomica> findById(Integer integer) throws Exception {
        return actividadEconomicaRepository.findById(integer);
    }

    @Override
    public void deleteById(Integer integer) throws Exception {
        actividadEconomicaRepository.deleteById(integer);
    }

	@Override
	public List<ActividadEconomica> fetchByProjectId(Integer id) throws Exception {
		return actividadEconomicaRepository.fetchByProjectId(id);
	}
}
